src/lib/swapper/swappers/ThorchainSwapper/utils/makeSwapMemo/makeSwapMemo.ts

```diff
@@ -77,7 +80,9 @@ export const makeSwapMemo: MakeSwapMemo = ({ buyAssetId, destinationAddress, lim
 
   const abbreviatedThorAssetId = abbreviateThorAssetId(fullThorAssetId)
 
-  const memo = `s:${abbreviatedThorAssetId}:${parsedDestinationAddress}:${limit}:${THORCHAIN_AFFILIATE_NAME}:${THORCHAIN_AFFILIATE_BIPS}`
+  const memo = `s:${abbreviatedThorAssetId}:${parsedDestinationAddress}:${limit}:${THORCHAIN_AFFILIATE_NAME}:${
+    affiliateBps ?? '0'
```

Comment:
Ditto we `getOrDefault` affiliate  bps so shouldn't need to short circuit to `'0'` since it's already the default?

Reply:
You're right! Fixed in https://github.com/shapeshift/web/pull/4353/commits/4515e57f538f78e07970aec65f275bf40bea9d36.